packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-container.jsx

```diff
@@ -176,7 +176,15 @@ const ProofOfIdentityContainer = ({
                 />
             );
         case service_code.manual:
-            return <Unsupported manual={manual} />;
```

Comment:
As Habib mentioned we can use object literal here too. 
cause switch/case is not a efficient way of writing conditional codes